pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -140,7 +141,8 @@ type monConfig struct {
 	// PublicIP is the IP of the mon's service that the mon will receive connections on
 	PublicIP string
 	// Port is the port on which the mon will listen for connections
-	Port int32
+	Port     int32
+	PortName string
```

Comment:
This PortName isn't used anywhere now, right?

Reply:
Thank you I'll remove this